Assemble.c

```diff
@@ -652,8 +652,9 @@ static int load_devices(struct devs *devices, char *devmap,
 			/* prepare useful information in info structures */
 			struct stat stb2;
 			int err;
-			fstat(mdfd, &stb2);
-
+			if (fstat(mdfd, &stb2) != 0) {
+				goto error;
+			}
```

Comment:
You don't need braces in this case, there is one instruction inside if but it is not an issue, just FYI,